examples/rust/components/http-client/README.md

```diff
@@ -0,0 +1,38 @@
+# HTTP Hello World
+
+This is a simple Rust Wasm example that responds with a "Hello World" message for each request.
+
+## Prerequisites
+
+- `cargo` 1.75
+- [`wash`](https://wasmcloud.com/docs/installation) 0.27.0
+- `wasmtime` 19.0.0 (if running with wasmtime)
```

Comment:
Does the example not work with `20.x`?

Reply:
This is copy/pasta from http-hello. I can change